main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as source_file:
+        players_data = json.load(source_file)
+
+    for player in players_data:
+        guild_ = None
+        player_guild = players_data[player]["guild"]
+        if player_guild:
+            if not Guild.objects.filter(name=player_guild["name"]).exists():
+                guild_ = Guild(
+                    name=player_guild["name"],
+                    description=player_guild["description"],
+                )
+                guild_.save()
+            else:
+                guild_ = Guild.objects.filter(name=player_guild["name"])[0]
+
+        race_ = None
+        player_race = players_data[player]["race"]
```

Comment:
As for me, these variable names are not clear enough

Reply:
How would you name these variables?